classes/autoptimizeConfig.php

```diff
@@ -194,7 +206,7 @@ public function show() {
 <td><?php
 	$AOstatArr=autoptimizeCache::stats(); 
 	$AOcacheSize=round($AOstatArr[1]/1024);
-	echo $AOstatArr[0].__(' files, totalling ','autoptimize').$AOcacheSize.__(' Kbytes (calculated at ','autoptimize').date("H:i e", $AOstatArr[2]).')';
+	echo $AOstatArr[0].' '.__(' files, totalling ','autoptimize').' '.$AOcacheSize.' '.__(' Kbytes (calculated at ','autoptimize').' '.date("H:i e", $AOstatArr[2]).')';
```

Comment:
The spaces are there already, in the translatable strings, so this is not needed.